docs/detections/detection-engine-intro.asciidoc

```diff
@@ -51,20 +51,26 @@ To make sure you can access Detections and manage rules, see
 
 [float]
 [[cold-tier-detections]]
-== Compatibility with cold and frozen tier nodes
+== Manage data in cold and frozen tiers
 
-Cold and frozen {ref}/data-tiers.html[data tiers] hold time series data that is only accessed occasionally. In {stack} version >=7.11.0, {elastic-sec} supports cold but not frozen tier data for the following {es} indices:
+Cold {ref}/data-tiers.html[data tiers] store time series data that's accessed infrequently and rarely updated, while frozen data tiers hold time series data that's accessed even less frequently and never updated. If you're automating searches across different data tiers using rules, consider the following best practices and limitations.
 
-* Index patterns specified in `securitySolution:defaultIndex`
-* Index patterns specified in the definitions of detection rules, except for indicator match rules 
-* Index patterns specified in the data sources selector on various {security-app} pages
+[float]
+[[best-practices-data-tiers]]
+=== Best practices 
+
+* **Retention in hot tier**: We recommend keeping data in the hot data tier ({ilm-cap} hot phase) for at least 24 hours. {ilm-cap} policies that move ingested data from the hot phase to another phase (for example, cold or frozen) in less than 24 hours may cause performance issues and/or rule execution errors.
+* **Replicas for mission-critical data**: Your data should have replicas if it must be highly available. Since frozen tiers don't support replicas, shard unavailability can cause partial rule run failures. Shard unavailability may be also encountered during or after {stack} upgrades. If this happens, you can <<manually-run-rules,manually rerun>> rules over the affected time period once the shards are available.
 
-{elastic-sec} does *NOT* support either cold or frozen tier data for the following {es} indices:
+[float]
+[[limitations-data-tiers]]
+=== Limitations
 
-* Index patterns controlled by {elastic-sec}, including alerts and list indices
-* Index patterns specified in the definition of indicator match rules 
+Data tiers are a powerful and useful tool. When using them, keep the following limitations in mind:
 
-Using either cold or frozen tier data for unsupported indices may result in detection rule timeouts and overall performance degradation.
+* {ilm-cap} policies for indices controlled by {elastic-sec}, including alerts and list indices, must not be modified.
+* Indicator match rule performance can be severely impacted by querying data in frozen tiers.
```

Comment:
From what it says above about replicas for mission critical data, it sounds like this might apply to any rule type. Is that wrong?

Reply:
Are you asking about performance implications? If so, yes, there are performance implications any time. you query frozen, but with Indicator Match, it is especially pronounced. 

Ideally, users won't need to think about what tier they are querying, but for now we do have to be aware of it.

Reply:
@yctercero can we actually removed this?

> * Indicator match rule performance can be severely impacted by querying data in frozen tiers.

In the ["Limited support for indicator match rules"](https://security-docs_bk_6659.docs-preview.app.elstc.co/guide/en/security/8.x/detection-engine-overview.html#support-indicator-rules) section, we make a similar statement:

> Indicator match rules don’t support cold or frozen data. Cold or frozen data in indices queried by indicator match rules must be older than the time range queried by the rule. If your data’s timestamps are unreliable, you can exclude cold and frozen tier data using a [Query DSL filter](https://security-docs_bk_6659.docs-preview.app.elstc.co/guide/en/security/8.x/exclude-cold-frozen-data-individual-rules.html).

Reply:
I'd be ok with that! 

cc @marshallmain @approksiu

Reply:
Yeah I think we can remove it

Reply:
```suggestion
```